.github/workflows/firesim-run-tests.yml

```diff
@@ -98,8 +98,15 @@ jobs:
     needs: change-filters
     if: needs.change-filters.outputs.needs-manager == 'true'
     runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash -el {0}
     steps:
       - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          environment-file: conda-reqs/ci-shared.yaml
+          miniforge-version: latest
       - name: Install Python CI requirements
```

Comment:
Do you still need to install python requirements here? This should be done by the conda incubator right.

Reply:
The requirements file is now moved to a conda syntax (the `ci-shared.yaml` file). This step installs those requirements into a conda environment and "activates" it by default in all the following shells